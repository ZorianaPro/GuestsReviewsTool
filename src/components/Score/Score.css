.Score {
  background-color: var(--colorBlue);
  display: inline-block;
  color: var(--colorWhite);
  padding: 0.25em 0.5em;
  border-radius: 0.5em;
  min-width: 4em;
  text-align: center;
  margin-right: 1em;
}

.Score > p {
  margin: 0 auto;
  font-size: var(--fontSize16);
}

.Score > p span {
  font-family: var(--fontFamilySFProBold);
}