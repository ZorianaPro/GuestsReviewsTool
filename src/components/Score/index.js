import React from 'react';
import './Score.css';

const Score = ({
  score,
  maximumScore = 5
}) => (
  <div className="Score">
    <p>
      <b>{ score }</b> / { maximumScore }
    </p>
  </div>
);

export default Score;