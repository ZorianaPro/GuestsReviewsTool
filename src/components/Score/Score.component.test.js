import React from 'react';
import { shallow, mount } from 'enzyme';
import Score from './';

describe('Score', () => {
  it('renders without crashing', () => {
    expect(
      shallow.bind(
        shallow,
        <Score/>
      ))
      .not
      .toThrow();
  });

  it('renders as expected', () => {
    expect(
      mount(
        <Score />
      )
    ).toMatchSnapshot();
  });
});
