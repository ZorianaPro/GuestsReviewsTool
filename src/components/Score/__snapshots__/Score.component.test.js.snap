// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Score renders as expected 1`] = `
<Score>
  <div
    className="Score"
  >
    <p>
      <b />
       / 
      5
    </p>
  </div>
</Score>
`;
